ops/testing.py

```diff
@@ -233,6 +233,10 @@ def get_relation_data(self, relation_id, app_or_unit):
         """
         return self._backend._relation_data[relation_id].get(app_or_unit, None)
 
+    def get_workload_version(self):
```

Comment:
using the same pattern as get_relation_data.
Also, using workload version because that is what we want charmers to care about.